web/src/components/dashboard/SidebarCardFundingNudge.tsx

```diff
@@ -13,16 +23,104 @@ type Props = {
 
 export const SidebarCardFundingNudge = (props: Props): ReactElement => {
   const [modalOpen, setModalOpen] = useState<boolean>(false);
+  const [profileData, setProfileData] = useState<ProfileData>(createEmptyProfileData());
```

Comment:
pebble - if you were going to do this in a file where we also reference business?.profileData a bunch I would strongly suggest a more aggressive naming strategy to make it clear when you're using the temporary state variable as opposed to the global state variable. 

That being said I'm not sure what the purpose of this state variable is really? the Setter is only called on line 36 which I think implies that business?.profileData and this profileData variable should always be the same

Reply:
Renamed to `tempProfiledata` in both `SidebarCardFundingNudge.tsx` and `TaxAccessStepTwo.tsx`.